src/main/1.3/scala/com/holdenkarau/spark/testing/DataframeGenerator.scala

```diff
@@ -35,7 +35,7 @@ object DataframeGenerator {
    * @return Arbitrary DataFrames generator of the required schema.
    */
   def arbitraryDataFrameWithCustomFields(sqlContext: SQLContext, schema: StructType, minPartitions: Int = 1)
-                                        (userGenerators: ColumnGenerator*): Arbitrary[DataFrame] = {
+                                        (userGenerators: Any*): Arbitrary[DataFrame] = {
```

Comment:
Could we make a common base class or Either here instead? Trying to keep it more specific than Any for user facing inputs.